libraries/chain/fork_database.cpp

```diff
@@ -456,16 +519,21 @@ namespace eosio { namespace chain {
       }
 
       for( const auto& block_id : remove_queue ) {
-         auto itr = my->index.find( block_id );
-         if( itr != my->index.end() )
-            my->index.erase(itr);
+         auto itr = index.find( block_id );
+         if( itr != index.end() )
+            index.erase(itr);
```

Comment:
Can't we replace these 3 lines with `index.erase(block_id);`?

Reply:
Yes.